apps/iac/gke.sh

```
@@ -0,0 +1,24 @@
+gcloud beta container --project "platform-hostv3" clusters create "manual1" \
```

Shellcheck

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)